### ISSUE TITLE
Removal of notaries from the Fil+ program

### ISSUE THREAD

**Issue by lilisy90** [filecoin-project/notary-governance#901]
## Issue Description
Notaries signed applications with 0% HTTP Retrievals Success Ratio

## Proposed Solution(s)
Removal from the FIl+ program if community audit supports T&T WG Lead audit

kernelogic
NiwanDao
Protocan Labs
sxxfuture-official
Suyanj
a1991car
liyunzhi-666
Joss-Hua

### Timeline
Discussion at June 27th 2023 WG Call
Community review until June 30th 2023 at 5 pm ET
### Technical dependencies 
Removal from main multisig and suspend them from R5 election
## Risks and mitigations 
Datacap Abuse
## Related Issues

https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1662
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1876
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1652
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/2050
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/881
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1664
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1874
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1938
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1948
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1000
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1720
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1767
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1682
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1483

**Comment by kernelogic** [comment]
You call this 0% success? I appreciate your effort but this accusation is baseless.

https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1483#issuecomment-1597668895

**Comment by lilisy90** [comment]
> You call this 0% success? I appreciate your effort but this accusation is baseless.
> 
> [filecoin-project/filecoin-plus-large-datasets#1483 (comment)](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1483#issuecomment-1597668895)
> 
> <img alt="image" width="842" src="https://user-images.githubusercontent.com/3711825/249155294-f2e4611c-0728-4e0a-a363-3b9ef12f8913.png">

Look at HTTP retrieval success rate: 0.00%

**Comment by kernelogic** [comment]
Any of the 3 retrieval methods are valid. Lotus market module supports Graphsync, boost supports HTTP / Bitswap.

**Comment by liyunzhi-666** [comment]
About #1000 I contacted the client to do KYB/KYC, confirmed his identity and checked the data sample he prepared. So I supported the first round of this application. All this happened before retrieving the retrieval bot.Apart from that, I have been keeping track of all the applications that I have signed. I discovered the retrieval problem two days ago and have contacted the client on WeChat about it and he said he was contacting SP to resolve the issue.

About #1682. I signed it before the retrieval bot came along and there were no problems at that time. Now look, it's not below 1% retrieval as well.

About #1682. also before the retrieval bot came along. I signed it and had no problems at the time. Now, it's not below 1% either.
